fusillade/__init__.py

```diff
@@ -23,11 +26,17 @@
 
 '''
 
-directory_name = Config.get_directory_name()
-try:
-    directory = CloudDirectory.from_name(directory_name)
-except FusilladeException:
-    from .clouddirectory import publish_schema, create_directory
-    schema_name = Config.get_schema_name()
-    schema_arn = publish_schema(schema_name, version="0.1")  # TODO make version an environment variable
-    directory = create_directory(directory_name, schema_arn)
+
+def create_directory():
```

Comment:
Since you're importing and calling `create_directory` inside of a different `create_directory`, could it be given a different name?

Reply:
changing to `get_directory`